### PR TITLE
Create a new setting for kernel picker experiment, preserving the previous experiment setting

### PR DESCRIPTION
**Pull request by IanMatthewHuff**
Fixes #

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [x] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for feature-requests.
-   [x] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2008,6 +2008,17 @@
                     "tags": [
                         "experimental"
                     ]
+                },
+                "jupyter.experimental.kernelPickerType": {
+                    "type": "string",
+                    "enum": [
+                        "Stable",
+                        "OnlyOneTypeOfKernel",
+                        "Insiders"
```
Comment (DonJayamanne) [inline package.json:2017]: I think you can make use of `enumDescriptions` to add descriptions for each item, its a new VSC API.

**Review by DonJayamanne — APPROVED** [review]
If i'm not mistaken this PR doesn't add a new Kernel picker UI, but only prepares for that by adding a new setting (which currently doesn't do anything).
Is that correct?

Apart from that LGTM

File: package.json (hunk 1 above)
Comment (IanMatthewHuff) [reply package.json:2017]: Ohh, nice. Will change to use that.

**Comment by IanMatthewHuff** [comment]
> If i'm not mistaken this PR doesn't add a new Kernel picker UI, but only prepares for that by adding a new setting (which currently doesn't do anything). Is that correct?
> 
> Apart from that LGTM

@DonJayamanne Yup. Don't want to go too crazy with PRs, but since this had to tweak a number of files I liked having it separate from when the new picker UI experiment starts to drop. Makes new code more obvious to look at with the small fixup to the old setting done.

**Pull request merged**